### ISSUE TITLE
Add badges to README files

### ISSUE THREAD

**Issue by Omnikron13** [Omnikron13/go-utils#2]
Add the relevant useful/standard quick info badges to the README files.

 * Basic versions OK to go with current tooling
 - [x] LICENCE (this one is static, so _should_ be easy, but I'm still debating ClearPL...)
 - [ ] Build passing/failing (atm per sub-dir? so just maths..?)
 - [ ] Open issues - should probably split this into at least bugs/non-bugs, etc...
 - [x] dependabot
 * Waiting on go-devoptions tooling
   - [ ] Code coverage
   - [ ] docs link - will rely on custom tooling from go-devoptions for gh-pages version
 * Waiting on being meaningful
   - [ ] Latest release (there are variants of this - investigate)
   - [ ] Open PRs
   - [ ] go.dev link(s)

## Drafts

 * Tests/checks
 * Coverage
   - *NOTE:* this is just hardcoded at 100%
     Waiting on go-devoptions work really?
 * Something to alert for swings (positive or negative) either decent sized but isolated, or modest but broad?
 * latest commit/activity
   - NOTE: With some fancy tooling this could display info about commits, merges

### Sort for relation to CI?
[![Whether or not all automated tests passed on all targetted platforms](https://badgen.net/github/checks/Omnikron13/go-utils?labelColor=24273A&scale=1.5&color=a6d189)](https://github.com/Omnikron13/go-utils/maths)
[![% of statements automated testing reached at least once](https://badgen.net/static/coverage/100%25/a6d189?labelColor=24273A&scale=1.5)](https://github.com/Omnikron13/go-utils/maths)

### Sort for relation to interaction & activity?
![Last commit (?)](https://badgen.net/github/last-commit/Omnikron13/go-utils?labelColor=24273A&scale=1.5&color=a6d189)
![All open issues](https://badgen.net/github/open-issues/Omnikron13/go-utils?labelColor=24273A&scale=1.5&color=a6d189)

 * open issues 
   - NOTE: again, would be better tracking bug/non-bug/untriaged really
 * Pending need
 * Help wanted = not currently! 
 * Open PRs ![](https://badgen.net/github/open-prs/Omnikron13/go-utils?labelColor=24273A&scale=1.5)
 * Latest release - currently pretty pointless?

![](https://badgen.net/github/label-issues/Omnikron13/go-utils/help-wanted/open?labelColor=24273A&scale=1.5)
![](https://badgen.net/github/release/Omnikron13/go-utils/stable?labelColor=24273A&scale=1.5)
